docs/advanced.md

```diff
@@ -14,11 +14,11 @@ commands. tldr; default commands allow you to define the entry point to your
 application using a similar API to subcommands.
 
 ```js
-const argv = require('yargs/yargs')(process.argv.slice(2))
+const argv = require('yargs')(process.argv.slice(2))
   .command('$0', 'the default command', () => {}, (argv) => {
     console.log('this command will be run by default')
   })
-  .parse()
+  .parse(process.argv.slice(2))
```

Comment:
I suggest `process.argv.slice(2))` in either factory call, or in parse, but not both.

Reply:
I believe I've addressed this, please feel free to follow up with a docs PR if I've missed your meaning.